### PR TITLE
Near field setup in HFSS

### LINKED ISSUE #1997: Get Near field setups from Radiation
### Description of the feature

Get the Near field definitions like the Far fields

### Steps for implementing the feature

_No response_

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Near field setup in HFSS. This important feature was not implemented.

Close #1997

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -5080,6 +5104,279 @@ def insert_infinite_sphere(
...
+
+        .. note::
+           This method is not supported by HFSS ``EigenMode`` and ``CharacteristicMode`` solution types.
+
+        Parameters
+        ----------
+        line : str
+            Polyline name.
+        points : float, str, optional
+            Number of points. The default is ``1000``.
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:5350]:
```suggestion
            Number of points. The default value is ``1000``.
```

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -190,12 +191,22 @@ def field_setups(self):
...
         -------
-        List of :class:`pyaedt.modules.Boundary.FarFieldSetup`
+        List of :class:`pyaedt.modules.Boundary.FarFieldSetup` and :class:`pyaedt.modules.Boundary.NearFieldSetup`
         """
         if not self._field_setups:
             self._field_setups = self._get_rad_fields()
         return self._field_setups
 
+    @pyaedt_function_handler()
+    def field_setup_names(self):
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:201]: You can use a property here.
```suggestion
    @property
    def field_setup_names(self):
```

**Review by MaxJPRey — APPROVED** [review]
This looks great.

File: _unittest/test_20_HFSS.py [hunk 3]
```diff
@@ -1119,3 +1119,70 @@ def test_54_assign_symmetry(self):
...
+            y_start=0,
+            y_stop=180,
+            y_step=10,
+            angle_units="deg",
+            custom_radiation_faces=None,
+            custom_coordinate_system=None,
+            name=None,
+        )
+        bound.name = "Test_Sphere"
+        assert self.aedtapp.field_setup_names()[0] == bound.name
```
Comment (Samuelopez-ansys) [inline _unittest/test_20_HFSS.py:1141]:
```suggestion
        assert self.aedtapp.field_setup_names[0] == bound.name
```

**Pull request merged**